.snyk

```diff
@@ -0,0 +1,9 @@
+# Snyk (https://snyk.io) policy file, patches or ignores known vulnerabilities.
```

Comment:
Please remove the grpc-core dependency from integrationtest-service-framework. It's not needed and also delete this after.